README.md

```diff
@@ -128,8 +129,7 @@ func main() {
     s := server.NewMCPServer(
         "Calculator Demo",
         "1.0.0",
-        server.WithResourceCapabilities(true, true),
```

Comment:
This MCP server only has tools. No need for declaring resource capabilities.

See: https://modelcontextprotocol.io/specification/2025-03-26/server/resources#capabilities